js/client-mainmenu.js

```diff
@@ -1175,6 +1237,25 @@
 				teambuilder.selectFolder(teamFormat);
 			}
 		},
+		selectFolder: function (key) {
+			var folderNotExpanded = this.folderNotExpanded;
```

Comment:
it seems like this `var` is unnecessary - saving having to type `this.` doesn't buy all that much? (this doesnt perform a copy, youd need to `.slice(0)` for that)